create_framework/unit_testing.rst

```diff
@@ -12,7 +12,7 @@ using `PHPUnit`_. At first, install PHPUnit as a development dependency:
 
 .. code-block:: terminal
 
-    $ composer require --dev phpunit/phpunit
+    $ composer require --dev phpunit/phpunit:9.6
```

Comment:
Shouldn't this be `composer require --dev phpunit/phpunit:^9.6`?

Reply:
Yes you are right! Thanks

Reply:
Yes you are right, thanks!